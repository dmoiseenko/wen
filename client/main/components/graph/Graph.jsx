import React from 'react';

import Sizer from '../../../common/components/Sizer';
import Lattice from './lattice/Lattice';


export default function Graph() {
  return (
    <div className="card graph">
      <Sizer>
        <Lattice />
      </Sizer>
    </div>
  );
}
