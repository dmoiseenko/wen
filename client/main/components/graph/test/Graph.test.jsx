import React from 'react';
import { shallow } from 'enzyme';

import Graph from '../Graph';


it('should render', () => {
  const wrapper = shallow(
    <Graph />
  );

  expect(wrapper.node).toMatchSnapshot();
});
