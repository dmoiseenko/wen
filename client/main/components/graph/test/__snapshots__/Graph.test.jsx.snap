// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="card graph"
>
  <Sizer>
    <Lattice
      height={0}
      width={0}
    />
  </Sizer>
</div>
`;
